src/flint/fmpq_mpoly.jl

```diff
@@ -56,11 +56,7 @@ function is_gen(a::QQMPolyRingElem, i::Int)
 end
 
 function is_gen(a::QQMPolyRingElem)
-  n = nvars(parent(a))
-  for i in 1:n
-    is_gen(a, i) && return true
-  end
-  return false
+  return Bool(@ccall libflint.fmpq_mpoly_is_gen(a::Ref{QQMPolyRingElem}, (-1)::Int, a.parent::Ref{QQMPolyRing})::Cint)
```

Comment:
```suggestion
  return Bool(@ccall libflint.fmpq_mpoly_is_gen(a::Ref{QQMPolyRingElem}, (-1)::Int, parent(a)::Ref{QQMPolyRing})::Cint)
```

Reply:
Makes sense, I just copied/adapted the ccall from the two parameter version: https://github.com/Nemocas/Nemo.jl/blob/174b902be17e22161987520799544975fe528a0d/src/flint/fmpq_mpoly.jl#L55

Reply:
Should we change it there two, and maybe also remove the unused `R = parent(a)`?